### PR TITLE
#162054516: Enable admin create, update and delete floors

### PR DESCRIPTION
**Pull request by omobosteven**
#### What does this PR do?
- Adds create, update and delete floor mutation to enable an admin to create floors in all the office.

#### How should this be manually tested?
- Run the server
- Test the mutation at `localhost:5000/mrm`
- Run `createFloor` mutation with the parameter 'block_id'  and 'name', as  required parameters
- Run `updateFloor` mutation with the parameter 'floor_id' and 'name' as required parameters
- Run `deleteFloor` mutation with the parameter 'floor_id' as required parameters

#### What are the relevant pivotal tracker stories?
[#162054516](https://www.pivotaltracker.com/story/show/162054516)

#### Screenshots
1. Create floor
image

2. Try to create floor in another location
image

3. Try to create floor that already exist
image

4. Create floor without name
image

5. Update floor
image

6 Update floor with a duplicate name
image

7 Try to update floor that does not exist
image

8. Delete floor
image

9. Try to delete floor that does not exist
image

### CODE REVIEW

**Comment by omobosteven** [comment]
@3V0L , I have implemented the requested changes.

File: api/floor/models.py [hunk 1]
```diff
@@ -14,3 +14,7 @@ class Floor(Base, Utility):
     block = relationship('Block')
     rooms = relationship('Room', cascade="all, delete-orphan")
     wings = relationship('Wing', cascade="all, delete-orphan")
+
+    @validates('name')
+    def convert_capitalize(self, key, value):
+        return value.capitalize()
```
[the remark this answers is not in the document]
Comment (omobosteven) [reply api/floor/models.py]: yes, But when I used `title()` it returned `1St Floor` and not `1st Floor`.
That was the reason I used `capitalize()`.

[the remark this answers is not in the document]
Comment (lennykioko) [reply api/floor/models.py]: I have experienced the same issue. I think it is ok for us to just use `.capitalize()` and ensure that we remain consistent.

[the remark this answers is not in the document]
Comment (omobosteven) [reply api/floor/models.py]: Okay, I will make the changes.

**Pull request merged**